Source/Core/VideoBackends/OGL/main.cpp

```diff
@@ -151,7 +151,7 @@ void VideoBackend::Video_Prepare()
   g_sampler_cache = std::make_unique<SamplerCache>();
   Renderer::Init();
   TextureConverter::Init();
-  BoundingBox::Init();
+  BoundingBox::Init(Renderer::EFBToScaledXf(1), Renderer::EFBToScaledYf(1));
```

Comment:
This should use `g_renderer->` instead of `Renderer::` (#4999 will eliminate static renderer state)